devices/tuya.js

```diff
@@ -143,7 +143,8 @@ module.exports = [
         description: 'Smart air house keeper',
         fromZigbee: [fz.tuya_air_quality],
         toZigbee: [],
-        exposes: [e.temperature(), e.humidity(), e.co2(), e.voc(), e.formaldehyd().withUnit('ppm'), e.pm25()],
+        exposes: [e.temperature(), e.humidity(), e.co2(), e.voc(), e.formaldehyd().withUnit('ppm'),
+            e.pm25().withValueMin(0).withValueMax(0).withValueStep(1)],
```

Comment:
Is correct to have min and max values to zero?

Reply:
Thanks, that was a mistake. The Value is 0-999. I commited the correction right now.